packages/cms-cli/lib/createTemplatePrompt.js

```diff
@@ -0,0 +1,21 @@
+const inquirer = require('inquirer');
+
+const TEMPLATE_TYPE_PROMPT = {
+  type: 'list',
+  name: 'templateType',
+  message: 'Select the type of template to create',
+  default: 'PAGE',
+  choices: [
+    { name: 'PAGE', value: 'page-template' },
+    { name: 'EMAIL', value: 'email-template' },
+    { name: 'GLOBAL PARTIAL', value: 'global-partial' },
```

Comment:
Should we add "Standard partial"? I also think that we should not use all caps and instead use sentence case. In terms of design, I think we should think about these prompts like the new file wizard in the Design Manager and also probably consider adding additional types.

![image](https://user-images.githubusercontent.com/266036/87826410-d4b32e80-c846-11ea-9d66-18903be9e5ff.png)

Reply:
Yeah, will be simple to just account for all current template skeletons. I'll add those.

Reply:
I added _some_ of these. I opted not to include them all, mainly to keep the list short and manageable. If anyone feels strongly, I can add remaining system and membership pages but I'd rather figure out a better pattern for surfacing these first, personally.